index.html

```diff
@@ -212,6 +212,39 @@ <h1 class="about-section-h1">About Me</h1>
                 </ul>
             </div>
         </section>
+
+        <section class="form-section">
+            <h1 class="form-h1">I'm always<br>interested in hearing<br>about new projects,<br>so if you'd like to<br>chat please get in touch.</h1>
```

Comment:
I strongly recommend you only add one `h1` tag once per HTML file, in order to use semantic HTML tags.

- [ ] Replace `h1` tag to `hN` tag where `N` is a number between (2...6).

Reply:
Thank you. I have made changes.

Reply:
@Whoistolu has made the changes however it's my understanding that having multiple `h1` elements once per html file is fine as long as there under a different section elemnt. which in this case they are. the `h1` element is inside a `section` tag and that is semantically correct.